src/pudl/transform/ferc1.py

```diff
@@ -2674,6 +2722,28 @@ def normalize_metadata_xbrl(
         return eead_meta
 
 
+class UtilityPlantSummaryFerc1TableTransformer(Ferc1AbstractTableTransformer):
+    """Transformer class for :ref:`utility_plant_summary_ferc1` table."""
+
+    table_id: Ferc1TableId = Ferc1TableId.UTILITY_PLANT_SUMMARY_FERC1
+    has_unique_record_ids: bool = False
+
+    def normalize_metadata_xbrl(
+        self, xbrl_fact_names: list[str] | None
+    ) -> pd.DataFrame:
+        """Normalize the metadata from the XBRL taxonomy +."""
+        eead_meta = (
```

Comment:
Maybe use a generic name for this variable here and in the electric energy account disposition table, since it's just locally scoped and having a table-specific name here that doesn't match is a bit of a non-sequitur.

Reply:
oh yeah oops. really we should think about standardizing this default method but there is some differences with the plant in service table that I didn't want to suss out just yet bc i don't know how standard this one rename will be.